### DISCUSSION TITLE
VRoid SDK 0.1.1 フィードバック：HubModelDeserializer

### DISCUSSION THREAD

**Discussion by hiirohitoyo** [pixiv/vroid-sdk-developers#47] [General]
VRoid SDK 0.1.1 にて気づいたことがありましたので、フィードバックしたいと思います。

[VRoid SDK 0.1.0 preview フィードバック: ModelLoader](https://github.com/pixiv/vroid-sdk-developers/discussions/29)
の[a]にて指摘されていることと同様のことがHubModelDeserializer.LoadVRMFromBinaryでも起きているようです。
非推奨クラスであることは理解しておりますが、情報共有として書かせていただきます。

**Comment by nyamadan** [comment]
報告ありがとうございます。

こちらも同様のことが発生しそうですね、修正を検討いたします。
